### DISCUSSION TITLE
Meu mapos não está abrindo depois que atualizei. Atualização feita em 05/06/2023

### DISCUSSION THREAD

**Discussion by lucasronieles** [RamonSilva20/mapos#2145] [Q&A]
Meu mapos não está abrindo depois que atualizei. Atualização feita em 05/06/2023.
Tem alguma sugestão. Ele simplesmente não loga no meu usuário

**Comment by belavistanet** [comment]
Bom dia
Estou usando o PHP 8.2 em um servidor Debian + Apache
Esta é minha primeira instalação  encontrei uma difculdade no momento de finalizar a instalação.

quando clico em finalizar fica na mensagem: Por favor, espere.

alguém sabe me dizer se preciso alterar algum arquivo?
![mapos](https://github.com/RamonSilva20/mapos/assets/125577718/fe44e42e-8ee4-4df6-ac21-7cef9a3064f2)
